src/fmu/dataio/_metadata.py

```diff
@@ -109,8 +108,7 @@ def generate_export_metadata(
     dataio: ExportData,
     fmudata: FmuProvider | None = None,
     compute_md5: bool = True,
-    skip_null: bool = True,
-) -> dict:  # TODO! -> skip_null?
```

Comment:
I removed skip_null as an option altogether. It doesn't seem to been used, or have ever been possible to use -- it was always True.

Reply:
nice 👍